packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -196,12 +223,17 @@ function registerCommands(
   const forceSourceStatusLocalCmd = vscode.commands.registerCommand(
     'sfdx.force.source.status.local',
     forceSourceStatus,
-    { flag: '--local' }
+    { flag: SourceStatusFlags.Local }
```

Comment:
should we pull this and line 321 into constants to enforce type safety and stand consistent?